starters/next13-zustand-bulma/README.md

```diff
@@ -1,38 +1,117 @@
 
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.ts`.
+This Kit uses new Next13 features, such as [`AppDir` router](https://beta.nextjs.org/docs/routing/fundamentals#the-app-directory) and [server components](https://beta.nextjs.org/docs/rendering/server-and-client-components).
 
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
+We've chose to split out a `components` directory for interactive UI components, and a `store` directory for the Zustand stores used for both global and some component state management.
+
+This allows us to keep the `app` directory focused on routing, layouts, and loading data, while passing that data to the more interactive components with more complex needs.
+
+### Styling and Theme
+
+The project uses [Bulma](https://bulma.io/) as the main styling framework. Bulma offers a way to "theme" their components and utility classes by overriding their default SASS variables.
+
+Overrides are done in the `base-styles.scss` file, which is imported in the `src/app/global.scss` file. Bulma components and utility classes can be added to components as needed as they are included in the `global.scss` file, which in turn is imported in the root layout.
+
+You should imoprt the `base-styles.scss` file in any component `*.module.scss` file, as per the example in `app/counter-example.page.tsx` to have access to the [Bulma variables]() for color and spacing in your component.
+
+Bulma is added to Storybook using the [`@storybok/addon-styling` package](https://storybook.js.org/addons/@storybook/addon-styling) which is the recommended way to add unsupported style frameworks to Storybook.
+
+There is some code included to prevent Storybook's `Source` component from picking up styles from the Bulma `tag` component class. The explanation and solution can be found in [this pr](https://github.com/thisdot/starter.dev/pull/1186).
+
+### Font Loading
+
+It takes advantage of the new [`next/font` package](https://beta.nextjs.org/docs/optimizing/fonts) for optimal font loading, and uses the [`@storybook/addon-styling` package](https://storybook.js.org/addons/@storybook/addon-styling), to provide the same font loading functionality to Storybook.
+
+### Colocation in directories
+
+Sometimes called "feature folders", or "modlets", this kit follows the convetion of grouping related files together in a directory regardless of filetype or extension. This makes it easier to find related files, and makes it easier to move files around without breaking imports. We also use the `index.ts` file to export some of the modules in the directory, so that we can import them from the directory path instead of the individual files: this is sometimes called a "Barrel" file.
```

Comment:
```suggestion
Sometimes called "feature folders", or "modlets", this kit follows the convention of grouping related files together in a directory regardless of filetype or extension. This makes it easier to find related files, and makes it easier to move files around without breaking imports. We also use the `index.ts` file to export some of the modules in the directory, so that we can import them from the directory path instead of the individual files: this is sometimes called a "Barrel" file.
```

Reply:
fixed in 2d02ec6